tests/testthat/test-printing.R

```diff
@@ -758,12 +788,10 @@ test_that("Separators and wrapping work together with getter and setters", {
     "thing its          ",
     "so                 ",
     "  m            8   ",
-    "                   ",
```

Comment:
i suspect this going to lead a few breaking changes downstream

Reply:
It could be, but as we discussed, theres no way to fix an inconsistency without a breaking change, and fixing it in the other direction will, i suspect, break *much more* downstream code on both your side and ours